tests/py/dynamo/converters/harness.py

```diff
@@ -234,6 +234,10 @@ def generate_graph(
         # Combine with customized passes specific to any model
         if customized_passes:
             passes_list.extend(customized_passes)
+
+        if disable_passes:
+            passes_list = []
+
         fx_module, _ = aten_tracer.trace(mod, original_inputs)
```

Comment:
This is a good addition. So I am assuming we should ideally set `disable_passes` for the subsequent tests?